cylc/flow/xtriggers/suite_state.py

```diff
@@ -14,24 +14,59 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-"""xtrigger function to check a remote suite state.
-
-"""
-
 import os
 import sqlite3
-from cylc.flow.cycling.util import add_offset
+
 from cylc.flow.cfgspec.glbl_cfg import glbl_cfg
+from cylc.flow.cycling.util import add_offset
 from cylc.flow.dbstatecheck import CylcSuiteDBChecker
-from metomi.isodatetime.parsers import TimePointParser
+from isodatetime.parsers import TimePointParser
```

Comment:
Was it intentional to use the old package name?

Reply:
Er, no, well spotted.